@here/generator-harp.gl/generators/app/index.js

```diff
@@ -55,8 +55,6 @@ module.exports = class extends Generator {
         this.spawnCommandSync("npm", [
             "install",
             "--save-dev",
-            "copy-webpack-plugin",
-            "html-webpack-plugin",
```

Comment:
I don't thing this can be removed now. In current state of patch, `copy-webpack-plugin` and second one will be unresolved because @here/harp-mapview specifies them as `devDependencies`.

Reply:
Correct, will add a dependency on the harp-webpack-utils module instead..